frontend/src/container/AdvancedSearch.jsx

```diff
@@ -143,7 +144,7 @@ export class AdvancedSearch extends React.Component {
     }
   }
 
-  getSearchResult = async (searchObject, currentSearchLength, searchFieldList, appliedFilters) => {
+  getSearchResult = async (searchObject, currentSearchLength, searchFieldList, appliedFilters = {}) => {
```

Comment:
Isn't it possible to put the default values here so that a user of this function directly knows which filters are managed?
For example:
```suggestion
  getSearchResult = async (searchObject, currentSearchLength, searchFieldList, appliedFilters = { createdRange: {}, modifiedRange: {}, newestAuthoredContentRange: {}, searchFacets: {}}) => {
```